graylog2-server/src/main/java/org/graylog2/alerts/AlertScanner.java

```diff
@@ -82,22 +83,36 @@ protected Alert handleTriggeredCheckResult(AlertCondition.CheckResult result, St
         return alert;
     }
 
+    private void handleResolveAlert(Alert alert) {
+        alertService.resolveAlert(alert);
+        // TODO: Send resolve notifications
```

Comment:
I added the comment to "justify" the existence of that one-line method, which would not make much sense otherwise.